bitnami/rabbitmq/README.md

```diff
@@ -577,7 +579,6 @@ extraConfiguration: |-
   default_permissions.write = .*
 ```
 
-
 ## Troubleshooting
```

Comment:
Please add the upgrade notes

Reply:
This is not a major change, we only add a parameter here.

Reply:
That's right 👍